### DISCUSSION TITLE
Cant sync F2A code, always trow "invalid code"

### DISCUSSION THREAD

**Discussion by victordariovera** [firefly-iii/firefly-iii#4891] [Firefly III]
**Bug description**
I am running Firefly III version 5.5.12, and my problem is: Cant sync F2A code, always trow "invalid code".

**Steps to reproduce**
Go to Profile configuration, 2FA and activate it. Using QR Code with a F2A mobile app and obtain temporary code. Type code in form and click save, Firefly trow an error "invalid code"

![Captura de Pantalla 2021-07-01 a la(s) 23 24 44](https://user-images.githubusercontent.com/25089852/124211784-1bd01900-dac4-11eb-9d25-d19185dab721.png)

**Comment by JC5** [comment]
Often happens when there is a large time difference between the server and the client. If you go to `/debug`, what's the timestamp in the first text box?

**Reply by victordariovera** [reply]
Hi!

Date is ok, totally in sync server and client. Debug logs dont show too much info. And now, without confirmation firefly has activated MFA in my user! And my token its not valid. I'm searching how to disable MFA from DB or other way.

![Captura de Pantalla 2021-07-02 a la(s) 11 40 06](https://user-images.githubusercontent.com/25089852/124291209-b9f6ca00-db2a-11eb-9323-8c0f34ff53d9.png)
![Captura de Pantalla 2021-07-02 a la(s) 11 39 58](https://user-images.githubusercontent.com/25089852/124291220-bbc08d80-db2a-11eb-878d-9d5c1e3fee01.png)
![Captura de Pantalla 2021-07-02 a la(s) 11 44 07](https://user-images.githubusercontent.com/25089852/124291310-d561d500-db2a-11eb-9bc2-ba3d35f91d31.png)

**Reply by victordariovera** [reply]
I'd deleted mfa code in users table with 

update users set mfa_code = '' where id = 1

And I'd regained access to my account.
I'll continue searching for a solution for the original issue.
